test/facets/embalmer-facet.spec.ts

```diff
@@ -97,8 +98,8 @@ describe("Contract: EmbalmerFacet", () => {
         arweaveArchaeologist.address,
         recipient.address,
         resurrectionTime,
-        sarcoToken.address,
-        canBeTransfered
+        canBeTransfered,
```

Comment:
Misspelling here (minor since this is a test, doesn't really matter).

Reply:
Fixed.